website/blog/2022-12-01-release-0.10-blog.md

```diff
@@ -0,0 +1,113 @@
+
+This can be done in the ‘Containers’ or 'Pods' screen of the application by clicking the `Play Kubernetes YAML` button and playing the existing YAML file.
+
+**3. Kube context automatically reloaded when updated**
+
+One can now reload kube context when file is changed or if content is changing.
+
+Launch Podman Desktop and then try to change the current kube context either by using the tray icon or with `kubectl` tool and then try to use ‘Deploy to Kubernetes’ feature, the selected context should be displayed.
+
+## UI Improvements
+
+**1. Kebab menu**
+
+With the latest release, you will see significant changes with respect to the options and how they are displayed for images and containers. Previously, there was an array of buttons over each container or image component which has now been replaced with a ‘kebab’ menu. This is more user-friendly and keeps all options organized within a single component. By clicking the three dot button, you can open the menu and choose the option depending on your workflow.
+
+![kebab menu](img/podman-desktop-release-0.10/kebab-menu.png)
+
+**2. Containers**
+
+The list of containers now also displays the age of the container as well as the image used to start the container in separate columns alongside the container name.
+
+**3. Responsive Design**
+
+On account of the effort to improve the User Experience, we have made the UI more responsive to window size changes. This includes animations as well as consistent design across all sections of the application.
+
+**4. Registries**
+
+We have also updated the UI of the page where we add/update registries in ‘Settings’.
+
+![registries](img/podman-desktop-release-0.10/registries.png)
+
+**5. Font and text**
+
+Other improvements include making font colors across the application consistent. Significant efforts have been made to display information correctly whether they are container logs or other text information.
+
+## Major enhancements and fixes
+
+**1. More options to configure containers**
+
+Until now, we could only specify port binding when building images to start containers. With the latest release, it includes several options such as configuring volumes, environment variables, some more advanced options including restart policy and options regarding networking and security. You can also reuse a network when creating containers.
+
+![create container basic](img/podman-desktop-release-0.10/create-container-basic.png)
+
+![create container security](img/podman-desktop-release-0.10/create-container-security.png)
+
+**2. Notify Proxy Changes**
+
+Updates to the proxy are now propagated to the listeners.
+
+**3. Automate the creation of the  Podman Socket when installing from Flatpack**
+
+On Linux, installing Podman Desktop with flatpak will detect if the Podman socket exists or not and create it if necessary. That fixes an issue where Podman Desktop was starting but not detecting Podman properly.
```

Comment:
`if the Podman socket exists or not and create it if necessary`
it looks to complex with exists/or not / if

Also it was like the most problematic bug on Linux

We fixed a long time bug in the flatpak version where it's now detecting the podman socket the first time we launch Podman Desktop. Before, flatpak was not seeing the socket as the folder was non-existing before starting the app.

Reply:
> * we had videos here it's only showing images (we can't highlight some use-cases properly with just a screenshot)
> * the release is v0.10.0, not v0.10
> * I think it's lacking a lot of fixed issues on proxy, certificates, and more that looks like more important than typo fixing

Including videos were the first thing that I thought of but then I remembered from our previous discussions where we decided on avoiding video content as it can reduce the speed at which website loads. I can still proceed with adding sample videos from the PR.

Reply:
https://github.com/containers/podman-desktop/pull/943/commits/ed51bbcba5b04ecdae8a6406aabbd2040153097a